Backend/router/v1/user.js

```diff
@@ -13,15 +13,34 @@ router.all('/', (req, res) => {
 });
 
 router.post('/signup', (req, res) => {
+
   let hashPassword = crypto.createHash("sha512").update(req.body.password).digest("hex");
 
-  models.user.create({
-    email: req.body.email,
-    password: hashPassword,
+  models.user.findAll({
+    where: {
+      email: req.body.email
+    }
+
   }).then(result => {
-    console.log("everything is good");
-    res.send(result);
+    if (result == "") {
+
+      models.user.create({
+          email : req.body.email,
+          password : hashPassword,
+      }).then(result => {
+          console.log("< " + req.body.email + " > membership has been completed.");
```

Comment:
Is it a good idea to show the email in for in the console? I don't think we should display such info. Same for other console.log lines here and there.